sqlglot/expressions.py

```diff
@@ -6534,6 +6534,10 @@ class ParseJSON(Func):
     arg_types = {"this": True, "expression": False, "safe": False}
 
 
+class ParseDatetime(Func):
+    arg_types = {"this": True, "format": False, "timezone": False}
```

Comment:
Rename `"timezone"` to `"zone"`– the latter is more consistent with the codebase. Also, include a test for the 3 argument version.